bats/tests/containers/factory-reset-containerd-shims.bats

```diff
@@ -0,0 +1,27 @@
+load '../helpers/load'
+
+local_setup_file() {
+    RD_USE_RAMDISK=false # interferes with deleting $PATH_APP_HOME
+}
+
+@test 'factory reset' {
+    delete_all_snapshots
+    rm -rf "$PATH_CONTAINERD_SHIMS"
+    factory_reset
+    assert_not_exists "$PATH_APP_HOME"
+}
+
+@test 'factory reset will not remove any shims' {
+    mkdir -p "$PATH_CONTAINERD_SHIMS"
+    touch "${PATH_CONTAINERD_SHIMS}/containerd-shim-bogus-v1"
+    factory_reset
+    assert_exists "$PATH_CONTAINERD_SHIMS"
```

Comment:
Shouldn't we assert that `containerd-shim-bogus-v1` exists instead? After all, we care that the shim is there, not the containing directory.

Reply:
Yes, good point, changed.